src/test/java/org/jenkinsci/plugins/workflow/DSLTest.java

```diff
@@ -159,19 +159,39 @@ public void monomorphic() throws Exception {
         r.assertLogContains("First arg: one, second arg: two", r.assertBuildStatusSuccess(p.scheduleBuild2(0)));
     }
 
+    @Issue("JENKINS-29711")
+    @Test
+    public void monomorphicSymbol() throws Exception {
+        WorkflowJob p = r.jenkins.createProject(WorkflowJob.class, "monSymbol");
+        p.setDefinition(new CpsFlowDefinition("monomorphWithSymbolStep([firstArg:'one', secondArg:'two'])", true));
+        r.assertLogContains("First arg: one, second arg: two", r.assertBuildStatusSuccess(p.scheduleBuild2(0)));
+    }
+
+
     /**
      * Tests the ability to execute a step with an unnamed monomorphic list argument.
      */
     @Issue("JENKINS-29711")
     @Test
     public void monomorphicList() throws Exception {
-        WorkflowJob p = r.jenkins.createProject(WorkflowJob.class, "mon");
+        WorkflowJob p = r.jenkins.createProject(WorkflowJob.class, "monList");
         p.setDefinition(new CpsFlowDefinition("monomorphListStep([[firstArg:'one', secondArg:'two'], [firstArg:'three', secondArg:'four']])", true));
         WorkflowRun b = r.assertBuildStatusSuccess(p.scheduleBuild2(0));
         r.assertLogContains("First arg: one, second arg: two", b);
         r.assertLogContains("First arg: three, second arg: four", b);
     }
 
+    @Issue("JENKINS-29711")
+    @Test
+    public void monomorphicListWithSymbol() throws Exception {
+        WorkflowJob p = r.jenkins.createProject(WorkflowJob.class, "monListSymbol");
+        p.setDefinition(new CpsFlowDefinition("monomorphListSymbolStep([[firstArg:'one', secondArg:'two'], [firstArg:'three', secondArg:'four']])", true));
```

Comment:
Why are you testing the deprecated syntax? If there is a symbol, you should be using it.